### PR TITLE
Updating 4.5 versions from 4.5.0 to 4.5.2

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

File: modules/cluster-logging-deploy-cli.adoc [hunk 1]
```diff
@@ -158,14 +158,14 @@ oc get csv --all-namespaces
...
+kube-node-lease                                         elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
+kube-public                                             elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
+kube-system                                             elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
+openshift-apiserver-operator                            elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
+openshift-apiserver                                     elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
+openshift-authentication-operator                       elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
+openshift-authentication                                elasticsearch-operator.4.5.2-202006160135    Elasticsearch Operator   4.5.2-202006160135               Succeeded
```
Comment (mburke5678) [inline modules/cluster-logging-deploy-cli.adoc:168]: After installing 4.5.2 in cluster-bot, the Cluster Logging and Elasticsearch Operators are:
` clusterlogging.4.5.0-202007012112.p0           Cluster Logging          4.5.0-202007012112.p0              Succeeded`
`elasticsearch-operator.4.5.0-202007012112.p0   Elasticsearch Operator   4.5.0-202007012112.p0              Succeeded`

File: modules/cluster-logging-updating-logging.adoc [hunk 2]
```diff
@@ -76,7 +76,7 @@ The Cluster Logging Operator is shown as 4.5. For example:
 +
 ----
 Cluster Logging
-4.5.0-202005201915 provided
+4.5.2-202005201915 provided
```
Comment (mburke5678) [inline modules/cluster-logging-updating-logging.adoc:79]: 4.5.0-202007012112.p0

File: metering/metering-upgrading-metering.adoc [hunk 3]
```diff
@@ -47,7 +47,7 @@ The Metering Operator is shown as 4.5. For example:
 +
 ----
 Metering
-4.5.0-202005202015 provided by Red Hat, Inc
+4.5.2-202005202015 provided by Red Hat, Inc
```
Comment (mburke5678) [inline metering/metering-upgrading-metering.adoc:50]: 4.5.0-202007012112.p0

File: metering/metering-upgrading-metering.adoc [hunk 4]
```diff
@@ -73,7 +73,7 @@ In the following example, the 4.5 Metering Operator upgrade is successful and re
 +
 ----
 NAME                                        DISPLAY                  VERSION              REPLACES                               PHASE
-metering-operator.4.5.0-202006161654        Metering                 4.5.0-202006161654   metering-operator.4.4.0-202005252114   Succeeded
+metering-operator.4.5.2-202006161654        Metering                 4.5.2-202006161654   metering-operator.4.4.0-202005252114   Succeeded
```
Comment (mburke5678) [inline metering/metering-upgrading-metering.adoc:76]: metering-operator.4.5.0-202007012112.p0        Metering                 4.5.0-202007012112.p0              Succeeded

**Review by adellape — APPROVED** [review]
:ok_hand:

**Comment by mburke5678** [comment]
LGTM

**Comment by bergerhoffer** [comment]
@vikram-redhat Your email mentioned that 4.5.1 was the official GA version, but I thought that 4.5.2 was going to be it. Can you just confirm that this PR should be updated to 4.5.1 instead of 4.5.2? Thanks!

**Comment by vikram-redhat** [comment]
/hold till we know what is happening with 4.5.3.

@jeana-redhat @codyhoag

**Comment by vikram-redhat** [comment]
@bergerhoffer we should be able to update it after getting the right images. @codyhoag - could you help and validate?

**Comment by codyhoag** [comment]
@vikram-redhat @bergerhoffer So our official story for the 4.5 release: 4.5.1 is the official GA, but customers can only upgrade to 4.5.4 (releasing July 30). So however this PR was updated based on 4.5.2, it should be updated instead with 4.5.4.

4.5.1-4.5.3 did not allow customers to upgrade to them, so no one should be using them.

**Comment by bergerhoffer** [comment]
Thanks @codyhoag I've updated to 4.5.4. Mind taking another look?

**Review by codyhoag — COMMENTED** [review]
One small suggestion. Everything else LGTM!

File: modules/cli-administrator-other.adoc [hunk 5]
```diff
@@ -45,8 +45,8 @@ Manage various aspects of the {product-title} release process, such as viewing i
 .Example: Generate a changelog between two releases and save to `changelog.md`
 ----
 $ oc adm release info --changelog=/tmp/git \
-    quay.io/openshift-release-dev/ocp-release:4.5.0-rc.7 \
-    quay.io/openshift-release-dev/ocp-release:4.5.0 \
+    quay.io/openshift-release-dev/ocp-release:4.5.4-rc.7 \
+    quay.io/openshift-release-dev/ocp-release:4.5.4 \
```
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply modules/cli-administrator-other.adoc]: The more accurate, the better! I've updated the example, thanks for pointing this out!

**Comment by bergerhoffer** [comment]
Unless there is any additional feedback in the meantime, we'll plan on merging this once 4.5.4 goes out.

**Comment by vikram-redhat** [comment]
4.5.4 is out, merge it quick before there is another changes :)

**Comment by bergerhoffer** [comment]
@vikram-redhat Done! :smile:

**Pull request merged**